crmprtd/align.py

```diff
@@ -114,17 +115,15 @@ def create_station_and_history_entry(sesh, network_name, native_id, lat, lon):
 
     network = network.first()
     stn = Station(native_id=native_id, network=network)
-    with sesh.begin_nested():
-        sesh.add(stn)
+    sesh.add(stn)
```

Comment:
Tell me more about why you removed the nested transaction here and below...

Reply:
I was looking for an issue with the insertion and thought that perhaps I was using the nested transactions incorrectly.  I actually meant to change this back.  Just a question, does using the ```sesh.begin_nested()``` as a context manager commit upon ```__exit__()```? Or is the commit at the bottom of the method still neccessary?

Reply:
@rod-glover Can you comment on this?